pkg/webhook/webhook.go

```diff
@@ -111,5 +113,26 @@ func register(manager manager.Manager, server *webhook.Server) error {
 				Decoder: manager.GetAdmissionDecoder(),
 			},
 		},
+	}, &admission.Webhook{
+		Name:          constants.AcceleratorInjectorMutatorWebhookName,
+		FailurePolicy: &constants.WebhookFailurePolicy,
+		Type:          webhooktypes.WebhookTypeMutating,
+		Rules: []v1beta1.RuleWithOperations{{
+			Operations: []v1beta1.OperationType{
+				v1beta1.Create,
+				v1beta1.Update,
+			},
+			Rule: v1beta1.Rule{
+				APIGroups:   []string{v1.GroupName},
+				APIVersions: []string{v1.SchemeGroupVersion.Version},
+				Resources:   []string{"deployments"},
+			},
+		}},
+		Handlers: []admission.Handler{
+			&deployment.Mutator{
```

Comment:
I don't think we should mutate all deployment is cluster. At least, we should not mutate deployments which is not related with kfserving (maybe we can filter by labels)

Reply:
Agreed. I'll update this.

Reply:
I am afraid we have to do it in the handler so far.
But in long term, we expect that knative configuration can add new 'field' to support customize deployment fileds such as .Spec.NodeSelector and .Spec.Tolerations in this case. Have no idea if knative serving has such plan